### PR TITLE
check if skopeo is in ubuntu

### PR DESCRIPTION
**Pull request by ivanovSPvirtru**

### CODE REVIEW

**Review by mustyantsev — APPROVED** [review]

File: .github/workflows/abacus-playwright-tests.yml [hunk 1]
```diff
@@ -44,6 +44,12 @@ jobs:
           kubernetes-version: 1.24.1
       - name: Run tilt
         run: |-
+          export BACKEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/backend \
+          | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][-1])")
```
Comment (dmihalcik-virtru) [inline .github/workflows/abacus-playwright-tests.yml:48]: Is `jq` included in the GitHub action runner?
```suggestion
          | jq -r '.Tags[-1]'
```

File: .github/workflows/abacus-playwright-tests.yml [hunk 2]
```diff
@@ -44,6 +44,12 @@ jobs:
...
+          echo "latest Backend is: $BACKEND_LATEST_VERSION"
+          export FRONTEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/abacus \
+          | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][-1])")
+          echo "latest Frontend is: $FRONTEND_LATEST_VERSION"
```
Comment (dmihalcik-virtru) [inline .github/workflows/abacus-playwright-tests.yml:52]:
```suggestion
          echo "Testing Frontend [$FRONTEND_LATEST_VERSION]">>$GITHUB_STEP_SUMMARY
```

File: .github/workflows/abacus-playwright-tests.yml [hunk 3]
```diff
@@ -44,6 +44,12 @@ jobs:
           kubernetes-version: 1.24.1
       - name: Run tilt
         run: |-
+          export BACKEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/backend \
+          | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][-1])")
+          echo "latest Backend is: $BACKEND_LATEST_VERSION"
```
Comment (dmihalcik-virtru) [inline .github/workflows/abacus-playwright-tests.yml:49]:
```suggestion
          echo "Testing Backend [$BACKEND_LATEST_VERSION]">>$GITHUB_STEP_SUMMARY

```

File: .github/workflows/vulnerability.yml [hunk 4]
```diff
@@ -46,5 +46,11 @@ jobs:
           kubernetes-version: 1.24.1
       - name: Run tilt
         run: |-
+          export BACKEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/backend \
+          | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][-1])")
+          echo "latest Backend is: $BACKEND_LATEST_VERSION"
```
Comment (dmihalcik-virtru) [inline .github/workflows/vulnerability.yml:51]:
```suggestion
          echo "Testing Backend [$BACKEND_LATEST_VERSION]">>$GITHUB_STEP_SUMMARY 
```

File: .github/workflows/vulnerability.yml [hunk 5]
```diff
@@ -46,5 +46,11 @@ jobs:
...
+          export FRONTEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/abacus \
+          | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][-1])")
+          echo "latest Frontend is: $FRONTEND_LATEST_VERSION"
```
Comment (dmihalcik-virtru) [inline .github/workflows/vulnerability.yml:54]:
```suggestion
          echo "Testing Frontend [$FRONTEND_LATEST_VERSION]">>$GITHUB_STEP_SUMMARY
```

File: .github/workflows/xtest.yml [hunk 6]
```diff
@@ -57,5 +57,13 @@ jobs:
           kubernetes-version: 1.24.1
       - name: Run tilt
         run: |-
+          export BACKEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/backend \
+            | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][32])")
+          echo "latest Backend is: $BACKEND_LATEST_VERSION"
+          export FRONTEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/abacus \
+            | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][53])")
+          echo "latest Frontend is: $FRONTEND_LATEST_VERSION"
```
Comment (dmihalcik-virtru) [inline .github/workflows/xtest.yml:65]:
```suggestion
          echo "Testing Frontend [$FRONTEND_LATEST_VERSION]">>$GITHUB_STEP_SUMMARY
```

File: .github/workflows/xtest.yml [hunk 7]
```diff
@@ -57,5 +57,13 @@ jobs:
           kubernetes-version: 1.24.1
       - name: Run tilt
         run: |-
+          export BACKEND_LATEST_VERSION=$(skopeo list-tags docker://ghcr.io/opentdf/charts/backend \
+            | python3 -c "import sys, json; sys.stdout.write(json.load(sys.stdin)['Tags'][32])")
+          echo "latest Backend is: $BACKEND_LATEST_VERSION"
```
Comment (dmihalcik-virtru) [inline .github/workflows/xtest.yml:62]:
```suggestion
          echo "Testing Backend [$BACKEND_LATEST_VERSION]">>$GITHUB_STEP_SUMMARY
```

File: opentdf.Tiltfile [hunk 8]
```diff
@@ -11,8 +11,8 @@ min_tilt_version("0.30")
 EXTERNAL_URL = "http://localhost:65432"
 
 # Versions of things backend to pull (attributes, kas, etc)
-BACKEND_CHART_TAG = "1.1.0"
-FRONTEND_CHART_TAG = "0.0.0-sha-f7c9fa0"
+BACKEND_CHART_TAG = os.environ.get("BACKEND_LATEST_VERSION")
+FRONTEND_CHART_TAG = os.environ.get("FRONTEND_LATEST_VERSION")
```
Comment (dmihalcik-virtru) [inline opentdf.Tiltfile:15]:
```suggestion
BACKEND_CHART_TAG = os.environ.get("BACKEND_LATEST_VERSION", "1.1.0")
FRONTEND_CHART_TAG = os.environ.get("FRONTEND_LATEST_VERSION", "1.1.0")
```

**Review by dmihalcik-virtru — DISMISSED** [review]
Minor nits. An IDK if jq is available

File: .github/workflows/abacus-playwright-tests.yml (hunk 1 above)
Comment (ivanovSPvirtru) [reply .github/workflows/abacus-playwright-tests.yml:48]: Nope))). I was going for it at first too))

**Review by dmihalcik-virtru — APPROVED** [review]

**Pull request merged**